lib/src/consts.rs

```diff
@@ -264,36 +306,12 @@ pub enum Network {
     TaikoA7,
 }
 
-impl FromStr for Network {
-    type Err = anyhow::Error;
-
-    fn from_str(s: &str) -> Result<Self, Self::Err> {
-        match s.to_lowercase().as_str() {
-            "ethereum" => Ok(Network::Ethereum),
-            "holesky" => Ok(Network::Holesky),
-            "taiko_a7" => Ok(Network::TaikoA7),
-            #[allow(clippy::needless_return)]
-            _ => bail!("Unknown network"),
-        }
-    }
-}
-
-impl Display for Network {
-    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
-        f.write_str(match self {
-            Network::Ethereum => "ethereum",
-            Network::Holesky => "holesky",
-            Network::TaikoA7 => "taiko_a7",
-        })
-    }
-}
-
 impl Network {
-    pub fn is_taiko(&self) -> bool {
+    pub fn to_string(&self) -> String {
         match self {
-            Network::Ethereum => false,
-            Network::Holesky => false,
-            Network::TaikoA7 => true,
+            Network::Ethereum => "ethereum".to_string(),
+            Network::Holesky => "holesky".to_string(),
+            Network::TaikoA7 => "taiko_a7".to_string(),
```

Comment:
Why not use a `Display` `impl` for this? It would be more general (when printing/logging)

Reply:
Yes, trait base is better, using ToString instead Display as I need to call to_string().